templates/partials/summary/list-summary.html

```diff
@@ -1,41 +1,45 @@
 {% from "components/summary/_macro.njk" import onsSummary %}
 
-{% if list.editable %}
-  {{ onsSummary({
-    "withinQuestion": true,
-    "summaries": [
-      {
-        "groups": [
-          {
-            "rows": map_list_collector_config(
-              list.list_items,
-              "person",
-              _("Change"),
-              _("Change details for {item_name}"),
-              _("Remove"),
-              _("Remove {item_name}")
-            ),
-          }
-        ]
-      }
-    ],
-    "classes": "u-mt-s u-mb-m"
-  }) }}
-{% else %}
-  {{ onsSummary({
-    "withinQuestion": true,
-    "summaries": [
-      {
-        "groups": [
-          {
-            "rows": map_list_collector_config(
-              list.list_items,
-              "person"
-            ),
-          }
-        ]
-      }
-    ],
-    "classes": "u-mt-s u-mb-m"
-  }) }}
+{% set display_summary = list.list_items | length > 0 %}
+
+{% if display_summary %}
+  {% if list.editable %}
+    {{ onsSummary({
+      "withinQuestion": true,
+      "summaries": [
+        {
+          "groups": [
+            {
+              "rows": map_list_collector_config(
+                list.list_items,
+                "person",
+                _("Change"),
+                _("Change details for {item_name}"),
+                _("Remove"),
+                _("Remove {item_name}")
+              ),
+            }
+          ]
+        }
+      ],
+      "classes": "u-mt-s u-mb-m"
+    }) }}
+  {% else %}
+    {{ onsSummary({
+      "withinQuestion": true,
+      "summaries": [
+        {
+          "groups": [
+            {
+              "rows": map_list_collector_config(
+                list.list_items,
+                "person"
+              ),
+            }
+          ]
+        }
+      ],
+      "classes": "u-mt-s u-mb-m"
+    }) }}
```

Comment:
We could remove some duplication here by assigning a `rows` variable dependent on `list.editable`, and calling `onsSummary` in one place.

Reply:
done